src/pages/search/search.module.scss

```diff
@@ -0,0 +1,30 @@
+@import "variables";
+
+.section {
```

Comment:
Do you use this section class somewhere?

Reply:
Wow, well spotted 🙂